bin/dfx-state-install

```diff
@@ -17,7 +17,7 @@ help_text() {
 # Source the clap.bash file ---------------------------------------------------
 source "$SOURCE_DIR/clap.bash"
 # Define options
-clap.define short=f long=state desc="The state file to save to" variable=DFX_STATE_FILE default="state.zip"
+clap.define short=f long=state desc="The state file to save to" variable=DFX_STATE_FILE default="$(dfx cache show)/state.zip"
```

Comment:
Can we name it snsdemo-state.zip or something like that so it's clear where this came from.
Otherwise when you find this file in the cache directory it will be unclear what it is and it's also more likely to have a collision with something unrelated.
And it will also make it easier to search for the related scripts.

Reply:
Yes, good idea.  Probably best done in a dedicated PR.  Small but touches several files.